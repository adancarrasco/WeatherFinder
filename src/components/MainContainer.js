import React from "react";

import styles from "./MainContainer.module.css";

const MainContainer = () => {
  return <div className={styles.mainContainer} />;
};

export default MainContainer;
