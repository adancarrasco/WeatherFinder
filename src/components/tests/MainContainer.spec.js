import React from "react";
import { shallow } from "enzyme";

import MainContainer from "../MainContainer";

describe("MainContainer", () => {
  it("it should render correctly with children", () => {
    const wrapper = shallow(
      <MainContainer>
        <div />
      </MainContainer>
    );
    expect(wrapper.debug()).toMatchSnapshot();
  });
});
