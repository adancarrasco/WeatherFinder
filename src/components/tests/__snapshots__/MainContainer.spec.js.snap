// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`MainContainer it should render correctly 1`] = `
"<div className=\\"mainContainer\\">
  <div />
</div>"
`;

exports[`MainContainer it should render correctly with children 1`] = `
"<div className=\\"mainContainer\\">
  <div />
</div>"
`;
